Gemfile

```diff
@@ -13,20 +13,21 @@ gem 'jbuilder',       '~> 2.10'
 gem 'jquery-rails'
 gem 'mini_magick',    '~> 4.10'
 gem 'puma'
-# gem 'passenger',      '>= 6.0.2', require: 'phusion_passenger/rack_handler'
+gem 'pg'
+
+gem 'bootstrap-sass'
+gem 'sassc-rails'
+gem 'uglifier'
```

Comment:
So would you prefer to compile the assets during the development phase and just copy them over?

Reply:
In my experience, it was always the case that the entire deployment process took a long time. Especially if it had to be repeated many times during troubleshooting. Even the whole nodejs(!) stuff had to be installed beforehand just for the asset compile. I just found it really ugly. Basically, I just think it's better if only the most necessary machinery is installed on production machines. Do you know what i mean? Do you have a more practical solution? I'm always open.

Reply:
On the other hand, we can also postpone such optimizations to a later point in time. We just shouldn't lose sight of it. :)

Reply:
I think some of the changes I have made already optimize it quite a bit especially for development. Building the image for production purpose should not happen as frequent and therefore can take longer. I restored these gems here because they are required by asset precompile in production mode. This only happens when you run `make deploy` otherwise this is not run. I hope this is making sense

Reply:
Okay, that's fine.